### PR TITLE
[en] config-pod-configmap add cleanup step and fix commands.

### PR DESCRIPTION
**Pull request by 1000Delta**
Change:

```
content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md
```

* Remove special-config before next step will create newer with different data.
* Fix quotation mark of cleanup command.
* Add cleanup command of created files and directories.

### CODE REVIEW

**Comment by 1000Delta** [comment]
See [Preview](https://deploy-preview-44094--kubernetes-io-main-staging.netlify.app/docs/tasks/configure-pod-container/configure-pod-configmap/).

File: content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 1]
```diff
@@ -910,7 +911,7 @@ kubectl delete pod dapi-test-pod --now
 
 # You might already have removed the next set
 kubectl delete configmaps/special-config configmaps/env-config
-kubectl delete configmap -l 'game-config in (config-4,config-5)’
+kubectl delete configmap -l 'game-config in (config-4,config-5)'
```
Comment (T-Lakshmi) [inline content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md:914]: Similar changes also require to do in line number 529. Please also update that.

Comment (T-Lakshmi) [reply content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md:914]: @1000Delta,
Please verify throughout the page if there is anything else to improve and update all necessary changes.

**Comment by 1000Delta** [comment]
I add commands in clean up section to:

1. remove `kustomization.yaml` to create configMap from generator
2. remove dir `configure-pod-container`

**Comment by T-Lakshmi** [comment]
[Preview page](https://deploy-preview-44094--kubernetes-io-main-staging.netlify.app/docs/tasks/configure-pod-container/configure-pod-configmap/#cleaning-up)

Looking Good To Me,
Thanks @1000Delta.

File: content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 2]
```diff
@@ -910,12 +911,23 @@ kubectl delete pod dapi-test-pod --now
...
+
+```bash
+rm kustomization.yaml
 ```
 
 If you created a directory `configure-pod-container` and no longer need it, you should remove that too,
 or move it into the trash can / deleted files location.
 
+```bash
+# Confirm that you are no longer need files in the directory, then remove it:
```
Comment (kbhawkey) [inline content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md:927]: nit: Do you need this comment? The previous sentence provides the same information.

File: content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 3]
```diff
@@ -584,6 +584,7 @@ Here is the manifest you will use:
 
   Once you're happy to move on, delete that Pod:
   ```shell
+  kubectl delete configmap special-config
```
Comment (kbhawkey) [inline content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md:587]: nit: Is this cleanup required? The comment before it is `delete that Pod`.

File: content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 4]
```diff
@@ -913,9 +914,19 @@ kubectl delete configmaps/special-config configmaps/env-config
...
+```bash
+rm kustomization.yaml
+```
+
 If you created a directory `configure-pod-container` and no longer need it, you should remove that too,
 or move it into the trash can / deleted files location.
 
+```bash
+rm -r configure-pod-container
+```
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md:928]: I'd remove `special-config` here too.

**Review by sftim — COMMENTED** [review]
@1000Delta, would you like to work on this further?

File: content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md (hunk 4 above)
Comment (sftim) [reply content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md:928]: Ah, cool.

**Comment by sftim** [comment]
@1000Delta, do you judge that this PR is now good to merge?

**Comment by 1000Delta** [comment]
Yes, I think this PR is ready to merge.

**Comment by 1000Delta** [comment]
/remove-lifecycle rotten

Hi, do I need more works about this PR or it's ready to merge now?

File: content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 5]
```diff
@@ -582,9 +582,10 @@ Here is the manifest you will use:
 
   Now, the Pod's output includes environment variables `SPECIAL_LEVEL_KEY=very` and `LOG_LEVEL=INFO`.
 
-  Once you're happy to move on, delete that Pod:
+  Once you're happy to move on, delete that Pod and ConfigMap:
   ```shell
   kubectl delete pod dapi-test-pod --now
+  kubectl delete configmap special-config
```
Comment (network-charles) [inline content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md:588]:
```suggestion
  kubectl delete configmap special-config
  kubectl delete configmap env-config
```

File: content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md (hunk 1 above)
Comment (network-charles) [reply content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md:914]: Done

File: content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 6]
```diff
@@ -913,9 +914,19 @@ kubectl delete configmaps/special-config configmaps/env-config
 kubectl delete configmap -l 'game-config in (config-4,config-5)'
 ```
 
+Remove the `kustomization.yaml` that you used to create the ConfigMap from generator:
```
Comment (network-charles) [inline content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md:917]: Improved clarity

```suggestion
Remove the `kustomization.yaml` file that you used to generate the ConfigMap:
```

**Pull request merged**